src/js/component/home.jsx

```diff
@@ -1,26 +1,78 @@
-import React from "react";
+import React, { useState } from "react";
+import {
+  materialRenderers,
+  materialCells,
+} from "@jsonforms/material-renderers";
+import { JsonForms } from "@jsonforms/react";
+import RatingControl, { ratingControlTester } from "../renderers/RatingControl";
```

Comment:
The `RatingControl` name is for respect the initial tutorial, but its content is not relevant